index.html

```diff
@@ -83,7 +83,7 @@
         </ul>
       </nav>
     </div>
-    <img class="section-bottom-tear" src="assets/section-bottom-tear-black.png" style="top:93%;">
+    <div class="page-tear"></div>
```

Comment:
Looks ok in most sections but the header is def weird with the texture showing up on top.